dev-qt/assistant/assistant-5.9.2.ebuild

```diff
@@ -29,6 +29,10 @@ QT5_TARGET_SUBDIRS=(
 	src/assistant/assistant
 )
 
+pkg_setup() {
+	use examples && QT5_EXAMPLES_SUBDIRS=("examples/assistant")
```

Comment:
I'm not happy about this. There's a sensible default that we can set in the eclass (`"examples"`), only few packages will need to override the default. Moreover, the USE flag check can be done in the eclass too.

Reply:
I don't think it's good idea to have ("examples") as default value. If USE-flag check is implemented in eclass, then all qtbase-based ebuilds would try to build and install examples unless default value is overwritten with some non-default empty value. And while it looks a lot, actually around a half or less of qt5 packages would use that new default value.

Reply:
my main problem with this approach is not the default value (or lack thereof), but the fact that you have to set `QT5_EXAMPLES_SUBDIRS` in a phase function, due to the `use examples` check. And I wonder why, since that check is duplicated in the eclass.